README.md

```diff
@@ -107,6 +107,15 @@ need to unblock the pipeline by following the Buildkite run link in the PR and
 click on the corresponding
 [unblock pipeline button](https://buildkite.com/docs/pipelines/block-step).
 
+## Kani proofs
+We run Kani proofs for each PR and on merges to the main branch. The proofs
+check conformance of the virtio-queue implementation to requirements from the
+virtio specification. Different than a unit test, a proof is checked for every
+possible input thus enforcing conformance. This is a list of the existing
+conformance proofs:
+
+* Conformance to 2.7.7.2 section
```

Comment:
I'm not sure it's a good idea to keep a list of proofs, just because I'm pretty sure we will forget to update xD.